src/AssetMapper/SassPublicPathAssetPathResolver.php

```diff
@@ -30,6 +30,10 @@ public function resolvePublicPath(string $logicalPath): string
 
     public function getPublicFilesystemPath(): string
     {
+        if (!method_exists($this->decorator, 'getPublicFilesystemPath')) {
+            throw new \Exception('...');
```

Comment:
I think it still would be great to have a dev-friendly message here to avoid confusing